index.js

```diff
@@ -29,8 +29,8 @@ module.exports = packages
  * KOA application instance.
  */
 
-function packages (dir, arg, skip) {
-  var modules = find.in(dir).findDirectories()
+function packages (dir, arg, skip, recursive) {
+  var modules = recursive ? find.from(dir).findDirectories() : find.in(dir).findDirectories()
```

Comment:
Let's just make this the default and drop the `recursive` option. No reason not to make it recursive AFAIKT. If you can think of any issues with that, let me know, but seems fine.